### PR TITLE
Adjusting README over the repo

### PR DESCRIPTION
**Pull request by Ellerbach**
- Cleaning README.md to make sure all internal links point on a real file
- Markdownlint the files to respect as best as possible Markdown rules
- Remove links on .fzz files

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/iot/pull/1621)

### CODE REVIEW

File: src/devices/Arduino/README.md [hunk 1]
```diff
@@ -17,65 +17,67 @@ In order to get an Arduino board working with the PC, you need to Install the Ar
...
 ```csharp
 // Portname is "COM3", "COM4" on Windows, "/dev/ttyUSB0" or similar on linux
 Console.WriteLine($"Connecting to Arduino on {portName}");
 ArduinoBoard board = new ArduinoBoard("COM3", 115200);
 try
 {
-	board.LogMessages += BoardOnLogMessages; // Get log messages
-	Console.WriteLine($"Connection successful. Firmware version: {board.FirmwareVersion}, Builder: {board.FirmwareName}");
-	// Add code that uses the board here.
+ board.LogMessages += BoardOnLogMessages; // Get log messages
```
Comment (krwq) [inline src/devices/Arduino/README.md:52]: should these be 4 spaces indent?

File: src/devices/Buzzer/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Here's an example how you could use `Buzzer`.
 ```csharp
 using (Buzzer buzzer = new Buzzer(21)); // Initialize buzzer with software PWM connected to pin 21.
 {
-	buzzer.PlayTone(440, 1000); // Play tone with frequency 440 hertz for one second.
+ buzzer.PlayTone(440, 1000); // Play tone with frequency 440 hertz for one second.
```
Comment (krwq) [inline src/devices/Buzzer/README.md:26]: same question about the indent

File: src/devices/Buzzer/samples/README.md [hunk 3]
```diff
@@ -1,89 +0,0 @@
-﻿
-# Example of Alphabet song played using Buzzer
```
Comment (krwq) [inline src/devices/Buzzer/samples/README.md:2]: should this be moved to the main readme?

File: src/devices/Buzzer/README.md [hunk 4]
```diff
@@ -83,25 +83,25 @@ Constructor takes a single parameter type of `Buzzer`.
...
 Following example demonstrates how to create MelodyElement sequence and how to play it using MelodyPlayer:
 
 ```csharp
 IList<MelodyElement> sequence =  new  List<MelodyElement>()
 {
-	new  NoteElement(Note.C, Octave.Fourth, Duration.Quarter),
-	new  PauseElement(Duration.Quarter),
-	new  NoteElement(Note.C, Octave.Fourth, Duration.Quarter)
+ new  NoteElement(Note.C, Octave.Fourth, Duration.Quarter),
+ new  PauseElement(Duration.Quarter),
```
Comment (krwq) [inline src/devices/Buzzer/README.md:98]: indent? here and below

File: src/devices/Card/CreditCard/README.md [hunk 5]
```diff
@@ -61,7 +62,7 @@ static void ReadCreditCard(Pn532 pn532)
 
         Console.WriteLine("All Tags for the Credit Card:");
         DisplayTags(creditCard.Tags, 0);
-		// Display Log Entries
+  // Display Log Entries
```
Comment (krwq) [inline src/devices/Card/CreditCard/README.md:65]: indent

File: src/devices/CharacterLcd/README.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ Please make sure you are using the [latest Bindings nuget](https://github.com/do
 
 This binding has been tested with a variety of 16x2 and 20x4 displays both in 4bit and 8bit mode and via i2C adapters (such as on the CrowPi). It should work with any character LCD with a 5x8 size character. Common names are 1602LCD and 2004LCD. Also supports [Grove - LCD RGB Backlight](http://wiki.seeedstudio.com/Grove-LCD_RGB_Backlight/).
 
-- [Very complete tutorial]https://learn.adafruit.com/drive-a-16x2-lcd-directly-with-a-raspberry-pi/overview() on how to connect and work with one of these displays.
+- [Very complete tutorial]<https://learn.adafruit.com/drive-a-16x2-lcd-directly-with-a-raspberry-pi/overview>() on how to connect and work with one of these displays.
```
Comment (krwq) [inline src/devices/CharacterLcd/README.md:11]: I think this was indented to be `[fff](xxx)` - note the brackets just after

File: src/devices/Common/README.md [hunk 7]
```diff
@@ -16,24 +16,24 @@ Some of the functions include:
...
 
-Logging is disabled by default. To enable logging to the console in an application, you can use this code: 
+Logging is disabled by default. To enable logging to the console in an application, you can use this code:
 
 ```csharp
 using var loggerFactory = LoggerFactory.Create(builder =>
 {
-	builder
-		.AddConsole();
+ builder
```
Comment (krwq) [inline src/devices/Common/README.md:28]: indent

File: src/devices/DCMotor/README.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ This is a generic class to control any DC motor.
 DC motors are controlled by simply providing voltage on the inputs (inverted voltage inverts the direction).
 
 DC motors can be controlled with 1, 2 or 3 pins.
-Please refer to the [sample](samples/README.md) to see how to connect it.
+Please refer to the [sample](./samples/Program.cs) to see how to connect it.
```
Comment (krwq) [inline src/devices/DCMotor/README.md:8]: are we gonna copy .cs files over to the docs repo? if not I'd recommed to remove the link all together or just generically say `sample` or full link

File: src/devices/DCMotor/samples/README.md [hunk 9]
```diff
@@ -1,55 +0,0 @@
-# DC Motor Controller sample
-
-![schematics](dcmotor_bb.png)
```
Comment (krwq) [inline src/devices/DCMotor/samples/README.md:3]: consider moving these to the main README

File: src/devices/Hmc5883l/samples/README.md [hunk 10]
```diff
@@ -1,36 +0,0 @@
-# HMC5883L - Samples
-
-## Hardware Required
```
Comment (krwq) [inline src/devices/Hmc5883l/samples/README.md:3]: consider moving to the main file

File: src/devices/Mpu9250/README.md [hunk 11]
```diff
@@ -27,7 +27,7 @@ var mpui2CConnectionSettingmpus = new I2cConnectionSettings(1, Mpu9250.DefaultI2
 using Mpu9250 mpu9250 = new Mpu9250(I2cDevice.Create(mpui2CConnectionSettingmpus), i2CDeviceAk8963: I2cDevice.Create(new I2cConnectionSettings(1, Ak8963.DefaultI2cAddress)));
 ```
 
-You can find an example in the [sample](./samples/Mpu9250.sample.cs) directory. Usage is straightforward including the possibility to have a calibration for all sub sensors.
+You can find an example in the [sample](./samples/Program.cs) directory. Usage is straightforward including the possibility to have a calibration for all sub sensors.
```
Comment (krwq) [inline src/devices/Mpu9250/README.md:30]: same question about copying .cs files - consider removing this link

File: src/devices/UFireIse/README.md [hunk 12]
```diff
@@ -4,68 +4,72 @@ The μFire ISE Probe Interface is an I²C sensor that can read a pH probe. Attac
...
-	if (uFireOrp.TryMeasureOxidationReductionPotential(out ElectricPotential orp))
-	{
-	    Console.WriteLine("Eh:" + orp.Millivolts);
-	}
-	else
-	{
-		Console.WriteLine("Not possible to measure pH");
-	}
+ if (uFireOrp.TryMeasureOxidationReductionPotential(out ElectricPotential orp))
+ {
```
Comment (krwq) [inline src/devices/UFireIse/README.md:33]: indent

File: src/devices/UFireIse/README.md [hunk 13]
```diff
@@ -4,68 +4,72 @@ The μFire ISE Probe Interface is an I²C sensor that can read a pH probe. Attac
...
-	}
-	else
-	{
-		Console.WriteLine("Not possible to measure pH");
-	}
+ Console.WriteLine("mV:" + uFire_pH.Measure().Millivolts);
+
+ if (uFire_pH.TryMeasurepH(out float pH))
+ {
+  Console.WriteLine("pH:" + pH);
```
Comment (krwq) [inline src/devices/UFireIse/README.md:68]: indent

**Review by krwq — APPROVED** [review]
LGTM, my main comment is indent and possibly losing sample, feel free to merge once that's addressed

File: src/devices/Buzzer/README.md (hunk 2 above)
Comment (Ellerbach) [reply src/devices/Buzzer/README.md:26]: catching up all of them now to replace the old tab which has been replaced by simple space to be aligned with the rest with 4 spaces

File: src/devices/DCMotor/README.md (hunk 8 above)
Comment (Ellerbach) [reply src/devices/DCMotor/README.md:8]: the links are transformed into link on the repo. So like an external link and you'll arrive on the repo.

File: src/devices/DCMotor/samples/README.md (hunk 9 above)
Comment (Ellerbach) [reply src/devices/DCMotor/samples/README.md:3]: they all have been move. The README.md still in /samples were left overs.

File: src/devices/Mpu9250/README.md (hunk 11 above)
Comment (Ellerbach) [reply src/devices/Mpu9250/README.md:30]: They are transformed like this:
image

**Pull request merged**